website/client/src/components/tasks/user.vue

```diff
@@ -300,6 +300,11 @@
       }
     }
 
+    .tags-list {
+      max-height: 400px;
+      overflow-y: scroll;
```

Comment:
@hazwong does this work with `overflow-y: auto; `? so that the scrollbar doesn't appear if not necessary?

Reply:
Currently, the scrollbar only appears when the tags list reaches a height of greater than 400px. The scrollbar doesn't appear if it's not necessary.